.env.example

```
@@ -5,3 +5,7 @@ VITE_SUPABASE_URL=your_supabase_url
 VITE_SUPABASE_ANON_KEY=your_supabase_anon_key
 # Optional dev override — production default is hardcoded in src/api/googleAuth.ts
 VITE_GOOGLE_CLIENT_ID=your_google_oauth_web_client_id
+
+# IS Mendelu credentials for scripts/scrape-real-data.ts (localhost real-data dev)
+MENDELU_USER=your_is_username
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The MENDELU_USER key is duplicated

(DuplicatedKey)